src/MergePlugin.php

```diff
@@ -188,6 +194,19 @@ function ($files, $pattern) use ($required) {
         }
     }
 
+    protected function validatePath($path){
+        if(substr($path, 0, 4) === 'http'){
+            $headers = get_headers ( $path );
```

Comment:
This should probably make a HEAD request (see https://secure.php.net/get_headers#example-4627)